.evergreen/check-all.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+set -o errexit
+
+EVERGREEN_DIR=$(dirname "$0")
```

Comment:
Why'd you opt for using this rather than the `.evergreen` directory name directly?

Reply:
I added it just in case the user was running the script within the `.evergreen` directory.

But now that you mention it, it looks like `check-rustfmt.sh` breaks if you run it from `.evergreen` instead of from the root, so this might actually be redundant.

Now that you mention it, we should probably replace it with this:
```suggestion
EVERGREEN_DIR=".evergreen"
```

That way, if the user runs the script from the wrong folder, it will immediately fail.